services/web/server/src/simcore_service_webserver/login/handlers_change.py

```diff
@@ -46,23 +46,47 @@
 
 
 class ResetPasswordBody(InputSchema):
-    email: str
+    email: LowerCaseEmailStr
 
 
-@routes.post(f"/{API_VTAG}/auth/reset-password", name="auth_reset_password")
-@global_rate_limit_route(number_of_requests=10, interval_seconds=HOUR)
-async def submit_request_to_reset_password(request: web.Request):
-    """
-        1. confirm user exists
-        2. check user status
-        3. send email with link to reset password
-        4. user clicks confirmation link -> auth/confirmation/{} -> reset_password_allowed
-
-    Follows guidelines from [1]: https://postmarkapp.com/guides/password-reset-email-best-practices
-     - You would never want to confirm or deny the existence of an account with a given email or username.
-     - Expiration of link
-     - Support contact information
-     - Who requested the reset?
+@routes.post(f"/{API_VTAG}/auth/reset-password", name="initiate_reset_password")
+@global_rate_limit_route(
```

Comment:
This means that if a few users all at once try to rese their passwords some will not be able to do so.
Let' say we have a class of sturneds and half of them forget their password. Please thing about if this limit is too low or not.
you want to protect abouse against automated systems, which I agree. Should we maybe not have this filtered by source IP?

Reply:
@mrnicegyu11 this relates to the discussion we had yesterday about traeffik. I wonder whether we could find a good compromise. We can check offline. There is not urgency for this now.